src/handlers/comment.ts

```diff
@@ -10,7 +10,7 @@ const commandHandlers: { [k in AllowedCommand]: (context: Context, commandArgume
       return logger.debug("Not an comment event");
     }
     const payload = context.payload;
-    const sender = payload.sender.login;
+    const sender = payload.sender?.login as string;
```

Comment:
Isn't this casting undefined as string? Seems wrong.

Reply:
Afaik it will always have a sender it's just how the github types are

the alternative is to remove the optional chaining and casting and have it cause a runtime error when login is undefined or we use `""` which is the same as `"undefined"` anyway

Whichever you think is best as it's a messy situation either way

Reply:
```typescript
const sender = payload.sender?.login as (undefined | string);
```

I guess